docs/self-managed/identity/orchestration-identity/configuration.md

```diff
@@ -12,42 +12,46 @@ As a Spring Boot application, Camunda 8 supports any standard
 [Spring configuration](https://docs.spring.io/spring-boot/reference/features/external-config.html) method.
 :::
 
-The following variables apply globally to all components within the Camunda Orchestration core: Zeebe, Operate, Tasklist, and Identity.
+The following variables apply globally to all components within the Camunda Orchestration core: Zeebe, Operate,
+Tasklist, and Identity.
 
 :::note
-The default configuration available below is for Camunda 8 Run. Helm installations have API authentication and authorizations enabled by default.
+The default configuration available below is for Camunda 8 Run. Helm installations have API authentication and
+authorizations enabled by default.
 :::
 
-<!-- updates must be made to BOTH tables -->
+<!-- updates must be made to BOTH tables. Note that they are sorted in alphabetical order! -->
 <Tabs>
   <TabItem value="env" label="Environment variables" default>
 
 | Environment variable                                | Description                                                                                       | Default value (Camunda 8 Run) |
 | --------------------------------------------------- | ------------------------------------------------------------------------------------------------- | ----------------------------- |
-| `SPRING_PROFILES_ACTIVE`                            | **Note:** This property will be deprecated as additional authentication methods become available. | `consolidated-auth`           |
-| `CAMUNDA_SECURITY_AUTHENTICATION_METHOD`            | The authentication method to use.                                                                 | `basic`                       |
-| `CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTED-API`   | If the API can be used without authentication.                                                    | `true`                        |
 | `CAMUNDA_PERSISTENT_SESSIONS_ENABLED`               | Enables shared authentication between the Orchestration web applications (Operate and Tasklist).  | `true`                        |
+| `CAMUNDA_SECURITY_AUTHENTICATION_METHOD`            | The authentication method to use.                                                                 | `basic`                       |
```

Comment:
🤔 Should we link to some docs where all the possible authentication methods are listed? It might be a bit un-clear to users what values they can set here since only the default (`basic`) value is provided.

We can also note that these values will be extended in the future, as we add them.

Reply:
I propose keeping it as-is for now and extend this when oidc gets documented. We don't have a place to reference at this time.